Modules/Devices/test/Devices.Domain.Tests/Identities/DeletionProcessTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Devices.Domain.Aggregates.Tier;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.Tooling;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Devices.Domain.Tests.Identities;
+
+public class DeletionProcessTests : IDisposable
```

Comment:
It's because the tests should use the same interface as the real users of the code. And according to the DDD practices you shouldn't use Entities directly, but only via their Aggregates.